### PR TITLE
Feature account validation

### PR DESCRIPTION
**Pull request by krishnajainGKMIT**

### CODE REVIEW

File: controllers/accountController.js [hunk 1]
```diff
@@ -0,0 +1,3 @@
+async function account(req, res) {
+  res.status(200).send("welcome to account");
+}
```
Comment (krishnajainGKMIT) [inline controllers/accountController.js]: no exports

Comment (krishnajainGKMIT) [reply controllers/accountController.js]: done

File: controllers/accountController.js
Comment (paras2103) [inline controllers/accountController.js:1]: account.controller.js

File: routes/accountRoute.js
Comment (paras2103) [inline routes/accountRoute.js:1]: accounts.route.js

**Pull request merged**